crmsh/main.py

```diff
@@ -303,6 +303,8 @@ def compgen():
 
 def parse_options():
     opts, args = option_parser.parse_args()
+    if opts.debug:
+        print(utils.debug_timestamp())
```

Comment:
@liangxin1300 I am not familiar with crmsh coding, but sounds weird to me to have this print into a function that has the main purpose of parsing the command line options.

Reply:
yes, this is not the best place for printing the timestamp. Please move the timestamp printing out of the options parser to a more appropriate location :)